client/src/constants/filter_constant.js

```diff
@@ -0,0 +1,19 @@
+const IMPRESSIONS = [
+  { label: 'All' },
+  { label: '≤ 10K', end: 10000 },
+  { label: '10K - 100K', start: 10001, end: 100000 },
+  { label: '100K - 1M', start: 100001, end: 1000000 },
+  { label: '1M - 10M', start: 1000001, end: 10000000 },
+  { label: '> 10M', start: 10000001 },
+];
+
+const COST = [
+  { label: 'All' },
+  { label: '≤ $100', start: 0, end: 100 },
+  { label: '$100 - $1,000', start: 101, end: 1000 },
```

Comment:
Aren't these bounds inclusive? Will it break if the start/ends overlap?

Reply:
Since my start and end boundaries are all separated clearly, we won't run into that problem

Reply:
Misspoke. If the true range is 101 - 1000, it should be labeled that way.